.github/workflows/Tests.yml

```diff
@@ -21,5 +21,13 @@ concurrency:
 jobs:
   tests:
     name: "Tests"
+    strategy:
+      matrix:
+        version:
+          - "1"
+          - "lts"
```

Comment:
`lts` is useful but IMO one of the most important tests is `min`. It's just too easy to accidentally break the oldest supported Julia version if it is not tested.

Reply:
min should be lts for all of our libraries.

Reply:
No, it's not. `min` is 1.10.0 and `lts` is 1.10.5.